gm4_blast_furnaces/data/blast_furnaces/functions/processes.mcfunction

```diff
@@ -0,0 +1,14 @@
+# blast furnace processes
+# @s - blast furnace
+# called by main
```

Comment:
Liquid tanks, block compressors, trapped signs, enchantment extractors, etc. All use `process`, instead of `processes`. This module should be consistent with that.

Reply:
That's what I said but I couldn't rename files cause that broke the FTP.